mds_loadtest/README.txt

```diff
@@ -0,0 +1,24 @@
+This repo contains scripts for running load tests. The framework we are using is called Locust,
+which is an open source program written in Python. It's very simple to setup and get going,
+which is the main reason why we chose it.
+
+Runs on:
+Python 3
+
+To install requirements run:
+
+'pip install -r requirements.txt'
+
+Get a bearer token from the network console after successfully logging into the app.
+Add it to the constants.py file
+NOTE:  It will only last a day.
```

Comment:
Not sure how much is involved in this... but does Locust support signing in using our bdd-test-* account(s) to produce an auth token the way that we do with our functional tests?

Also, it would be best to create a gitignored config/env file with these values instead of a constants file, regardless of which route we go.